### ISSUE TITLE
[Component Update] Update control items and text input API to allow max-width management

### ISSUE THREAD

**Issue by paulinea** [Orange-OpenSource/ouds-ios#1211]
### Description

Add a boolean parameter to OudsSwitchItem, OudsCheckboxItem, OudsRadioButtonItem and OudsTextInput:

**true** Apply the max-width token to limit component's width in large area.
**false** The component occupies the entire width of its container.

FYI: We called this boolean `constrainedMaxWidth` on Android side.

### Motivation & Context

A user should be able to choose if a control item or a text input must be displayed to fill its container in width, or using a max-width to constrain its width on large screens.

### Types of change

- [ ] Add new component
- [ ] Update existing component

### Checklist for Core Team

- [ ] Tokens have been updated
- [ ] Documentations have been updated
- [ ] Unit tests have been updated
- [ ] UI tests have been updated
- [ ] Design System Toolbox has been updated
- [ ] Retrocompatibility has been checked, or broken API have been flagged as deprecated with a workaround

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765922908**
- Build tag: **8ba3d47**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1766009311**
- Build tag: **117a933**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/1.0.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **1.0.0**
- Build number: **1766065729**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
